index.html

```diff
@@ -1,21 +1,106 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <!-- Add a link to your custom CSS here -->
+    <link rel="stylesheet" href="css/main.css">
 
     <title>Responsive grid project</title>
 </head>
 
 <body>
+    <div class="contener">
```

Comment:
It might be a good idea to wrap this in a header tag and give a class of the header instead of the content class and div